tests/Controller/GalleryAdminControllerTest.php

```diff
@@ -121,13 +121,21 @@ private function configureSetCsrfToken($intention)
     private function configureSetFormTheme($formView, $formTheme)
     {
         $twig = $this->prophesize('\Twig_Environment');
-        $twigRenderer = $this->prophesize('Symfony\Bridge\Twig\Form\TwigRenderer');
+
+        // Remove this trick when bumping Symfony requirement to 3.4+
+        if (method_exists('Symfony\Bridge\Twig\Command\DebugCommand', 'getLoaderPaths')) {
```

Comment:
DebugCommand::class

Reply:
Spelling out fully-qualified class names as strings is done *all over the place* in those test classes.

Even if I change this particular instance, or all the lines I've touched, or even all instances in those particular method, the test class would actually more inconsistent style-wise.

Might I propose to leave it as is for now and update the code style in a separate PR instead?

Reply:
@hanzi how about you do it in a separate commit, before your changes?